lib/px/utils/pxclient.lua

```diff
@@ -315,6 +319,13 @@ function M.load(px_config)
             ngx_req_set_header('cookie', 'pxvid=' .. vid)
         end
 
+        local xff_header = ngx.req.get_headers()['X-Forwarded-For']
```

Comment:
why not to do this at `forward_to_perimeterx`

Reply:
because it's only relevant for xhr calls